spec/integration/admin/comments_spec.rb

```diff
@@ -89,4 +89,17 @@
       it_behaves_like 'a not found page'
     end
   end
+
+  context "When <xmp> tag is used in comment author's name" do
+    before do
+      @comment.update(name: '<xmp>')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.